Tasks/Maven/moduleanalysis.ts

```diff
@@ -0,0 +1,7 @@
+import ar = require('./analysisresult');
+// Data class for supporting Maven projects with more than one module
+export class ModuleAnalysis {
+    moduleName: string;
+    rootFolder: string;
+    analysisResults:any = {}; // To be used as a dictionary
+}
```

Comment:
"// a dictionary of _tool name_ -> AnalysisResult" would be better than "// To be used as a dictionary"